Index: concessioengine/src/main/java/net/nueca/concessioengine/fragments/BaseProductsFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- concessioengine/src/main/java/net/nueca/concessioengine/fragments/BaseProductsFragment.java	(revision e63ce29daf6698047081d7cad89fbb3833f9907a)
+++ concessioengine/src/main/java/net/nueca/concessioengine/fragments/BaseProductsFragment.java	(revision )
@@ -108,7 +108,6 @@
 
         boolean includeSearchKey = !searchKey.equals("");
         boolean includeCategory = !category.toLowerCase().equals("all");
-
         try {
             Where<Product, Integer> whereProducts = getHelper().getProducts().queryBuilder().where();
             whereProducts.isNull("status");
